services/converter-ms-sems/tests/test_SEMSoutput.py

```diff
@@ -1,49 +1,51 @@
+import tempfile
 from unittest.mock import patch
 
-import pytest, json, io, os
+import json, io, os
 
 from converter.SEMSoutput import process_tallies_file
+from converter import SEMSinput
 
 PARENT_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..')
 SAMPLE_FILES = os.path.join(PARENT_DIR, 'sample_files')
 
-TESTS = [
-    {
-        'election': 'general-election.json',
-        'cvrs': '10_cvrs.csv',
-        'tallies': '10_tallies.json',
-        'sems': '10_expected-sems-results.txt'
-    },
-    {
-        'election': '10_8-26-2020-expected-election.json',
-        'cvrs': '10_8-26-2020-cvrs.txt',
-        'tallies': '10_8-26-2020-tallies.json',
-        'sems': '10_8-26-2020-expected-sems-output.txt',
-    },
-    {
-        'election': '10_8-26-2020-expected-election.json',
-        'cvrs': '10_8-26-2020-b-cvrs.txt',
-        'tallies': '10_8-26-2020-b-tallies.json',
-        'sems': '10_8-26-2020-b-expected-sems-output.txt',
-    },
-    {
-        'election': 'electionPrimarySample.json',
-        'cvrs': 'election-primary-sample-cvrs.txt',
-        'tallies': 'election-primary-sample-tallies.json',
-        'sems': 'election-primary-expected-results.csv',
-    }
-]
-
 def get_sample_file(filename):
     return os.path.join(SAMPLE_FILES, filename)
 
-def test_general_results_from_tallies():
-    for test in TESTS:
-        result = process_tallies_file(
-            get_sample_file(test['election']),
-            get_sample_file(test['tallies']))
-
-        expected_result_file = open(get_sample_file(test['sems']), "rb")
-        expected_result = expected_result_file.read()
+def test_general_results_from_tallies(snapshot):
+    with tempfile.NamedTemporaryFile('w') as election_file:
+        election = SEMSinput.main(
+            get_sample_file('10_8-26-2020.txt'),
+            get_sample_file('10_CANDMAP_8-26-2020.txt')
+        )
+        election_file.write(election)
+
+        TESTS = [
+            {
+                'election': get_sample_file('general-election.json'),
+                'cvrs': '10_cvrs.csv',
+                'tallies': '10_tallies.json',
+            },
+            {
+                'election': election_file.name,
+                'cvrs': '10_8-26-2020-cvrs.txt',
```

Comment:
Where are we using these `cvrs` files?

Reply:
Uhh, that's weird. They were there before but also unused. I guess I can remove?

Reply:
If it doesn't break anything, I think so. The test files in are in our current CVR file format, which we'll be changing anyway, so no use keeping around.

I didn't think we convert directly from our CVRs - we convert from an overall tally when we export results - so I'm not sure what those are doing there.

Reply:
Maybe an artifact from an earlier version of the script. Removed in 2da763a53